connector/tests/src/test/java/org/oaebudt/edc/ManagementApiTransferTest.java

```diff
@@ -78,45 +90,90 @@ class ManagementApiTransferTest {
 
     @Order(3)
     @RegisterExtension
-    static final BeforeAllCallback CREATE_DATABASES = context -> {
-        PROVIDER_POSTGRESQL_EXTENSION.createDatabase(PROVIDER.getName());
-        CONSUMER_POSTGRESQL_EXTENSION.createDatabase(CONSUMER.getName());
-        PROVIDER_FC_POSTGRESQL_EXTENSION.createDatabase(PROVIDER_FC.getName());
-    };
+    static final PostgresEndToEndExtension IDENTITY_HUB_PROVIDER_POSTGRESQL_EXTENSION = new PostgresEndToEndExtension();
 
     @Order(4)
     @RegisterExtension
-    static final HashiCorpVaultEndToEndExtension VAULT_EXTENSION = new HashiCorpVaultEndToEndExtension();
+    static final PostgresEndToEndExtension IDENTITY_HUB_CONSUMER_POSTGRESQL_EXTENSION = new PostgresEndToEndExtension();
 
     @Order(5)
     @RegisterExtension
+    static final HashiCorpVaultEndToEndExtension VAULT_EXTENSION = new HashiCorpVaultEndToEndExtension();
+
+    @Order(6)
+    @RegisterExtension
     static final KeycloakEndToEndExtension KEYCLOAK_EXTENSION = new KeycloakEndToEndExtension();
 
+    @Order(7)
+    @RegisterExtension
+    static final NginxEndToEndEntension NGINX_EXTENSION = new NginxEndToEndEntension();
```

Comment:
if this extension representing the Issuer right? if so, let's call it so, because "nginx" is pretty vague